serve/pyproject.toml

```diff
@@ -10,6 +10,11 @@ fastapi = ">=0.103.1"
 pydantic = ">=1.8.0"
 prometheus-client = ">=0.18.0"
 outlines = "0.0.23"
+# PT model dep
+rpyc = "5.3.1"
+tiktoken = "0.5.2" # for Qwen
+# TODO vllm dep
```

Comment:
One caution here is that I heard other folks had difficulty in setting up the dependency for vllm-mixtral. If we can simplify the dependencies by selectively supporting the models, it would be nice but not sure if that is easy to achieve.

Reply:
The difficulty was only in installing megablocks, and the latest vLLM no longer depends on it.

If some vLLM models depend on a package that's hard to install, we can simply drop support for such model. In `torch_model.py`, supported models are explicitly listed and unsupported models won't run.